source/administration/telemetry.rst

```diff
@@ -134,7 +134,7 @@ Non-personally Identifiable Diagnostic Information, distinguished by end users a
   - *Permissions Discovery Diagnostics (Enterprise Edition Only):* Provides all the permissions configured for each role for the System Scheme and each Team Override Scheme created in the system. Scheme ID; Team Admin Permissions; Team User Permissions; Channel Admin Permissions; Channel User Permissions; Number of teams the scheme is associated with
   - *Group Discovery Diagnostics (Enterprise Edition Only):* Provides information related to AD/LDAP groups, including number of groups synced to Mattermost, teams and channels associated to groups, teams and channels synced with groups, and number of group members
   - *System Console Menu Discovery Diagnostics:* Clicks on the hamburger menu items of the System Console, including Administrator's Guide, Troubleshooting Forum, Commercial Support, About Mattermost, and clicks on the left-hand side navigation menu items
-
+  - *In Product Notices Diagnostics:* Notices viewed and the notices on which action button was clicked.   
```

Comment:
```suggestion
  - *In Product Notices Diagnostics:* Notices viewed, and the notices on which an action button was clicked.   
```

Reply:
@amyblais Thank you. I was just waiting for ES to check this, Glad you did